src/components/lco/PageBanner.tsx

```diff
@@ -8,14 +8,19 @@ export interface LCOProfileType {
   areaIdList: string[]
   /** Official name */
   name: string
-  /** IG Url */
-  instagram: string
   /** Official website */
   website: string
   /** Bad hardware report form url */
-  report?: string
+  report: string | null
```

Comment:
why couldn't we use `report?: string` which implicitly supports null, undefined or string?


if we use `report !== null` (equal signs) and report is `undefined` it will evaluates to true
 
```javascript
{
  report != null && // only 1 equal sign
            <a className='btn btn-primary btn-sm opacity-80' href={report} target='_blank' rel='noreferrer'>Report bad bolts</a>
}
```